src/app/script/app_theme_object.cpp

```diff
@@ -49,12 +49,8 @@ class Theme {
 };
 
 struct ThemeDimension {
-  Theme* theme;
-
-  ThemeDimension(Theme* theme) : theme(theme) { }
-
   int getById(const std::string& id) const {
-    return theme->getDimensionById(id);
+    return skin::SkinTheme::instance()->getDimensionById(id);
```

Comment:
There is problem here that I'm looking now, the old `Theme*` pointer was pointing to the `class Theme` defined above (in this same anonymous namespace, which contains the `m_uiscale` to automatically adjust the dimension depending on the current uiscale added in #3813), it was using the `getDimensionById()` of that Theme (and not from ui::Theme).

We need other kind of fix for this.